packages/service/lib/notificationService.ts

```diff
@@ -53,7 +53,7 @@ export class NotificationService {
   async notifyNewRequest(request: Request) {
     const subject = "New Request";
 
-    const instructors = await this.services.user.getUsersFromClass(
+    const instructors = await this.services.user._getUsersFromClassInternal(
```

Comment:
Oh no, this is cursed. I knew we shouldn't do OOP in JavaScript...

Just a note here that we should probably change the services into functional modules in the future, so that a mixin trait can be used to provide something similar to a _protected_ access...